modules/verify-acs-installation.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * dir/filename.adoc
+:_module-type: CONCEPT
+[id="verify-acs-installation_{context}"]
+= Verify installation
+
+After you complete the installation and configure necessary integrations, run sample applications to evaluate the results of security assessments and policy violations.
+
+The sample applications described below are specifically designed to verify the build and deploy-time assessment features of {product-title}.
+
+To run the sample application:
+
+. Create a new namespace (or project).
++
+[source,bash]
+----
+$ oc new-project test
```

Comment:
Lets keep this OpenShift agnostic and go back to creating namespaces. We can't forget this won't only be installed on OpenShift.